multiple-select.js

```diff
@@ -480,11 +480,13 @@
         setSelects: function (values) {
             var that = this;
             this.$selectItems.prop('checked', false);
+            this.$disableItems.prop('checked', false);
             $.each(values, function (i, value) {
                 that.$selectItems.filter(sprintf('[value="%s"]', value)).prop('checked', true);
+                that.$disableItems.filter(sprintf('[value="%s"]', value)).prop('checked', true);
             });
             this.$selectAll.prop('checked', this.$selectItems.length ===
-                this.$selectItems.filter(':checked').length);
+                this.$selectItems.filter(':checked').length + this.$disableItems.filter(':checked').length));
```

Comment:
I test your code in jsfiddle: http://jsfiddle.net/wenyi/knbdm5hu/28/, but it cause an error.
this.$disableItems.filter(':checked').length**)**); --> error here

Can you update the jsfiddle to test your code and show the problem you want to solve?